daemon/src/model/cfd.rs

```diff
@@ -317,7 +317,7 @@ pub enum CfdEvent {
     ///
     /// This can happen in cases where we publish a settlement transaction while the lock
     /// transaction is still pending and they end up in the same block.
-    LockConfirmedAfterFinality,
+    LockConfirmedAfterClose,
```

Comment:
would changing the existing event cause a lot of disruptions at this stage? perhaps we should append a new one instead of changing this one?

Reply:
That's a very good point!

Reply:
I opted for keeping the old name `LockConfirmedAfterFinality` which is not entirely correct, but I think acceptable. I extended the doc comment.